week-2-wide-perspective/w02e05-simple-server-side/src/main.ts

```diff
@@ -0,0 +1,17 @@
+import 'express-async-errors'
+import express from 'express'
+import { logger } from './logger.js'
+import { router } from './router.js'
+import { errorHandler } from './error.handler.js'
+
+const app = express()
+
+app.use(express.json())
+app.use(router)
+app.use(errorHandler)
+
+app.listen(4000)
```

Comment:
Tutaj zrobiłbym `4000` jako `process.env.PORT` i jakiś default.

Dodatkowo cenna jest informacja przy starcie, na jakim porcie nasłuchuje serwer, np. wrzucając jego link.
Możesz to osiągnąc za pomocą 2 argumentu jako callback dla `listen(x, () => {})`